src/modules/job-manager/plugins/perilog.c

```diff
@@ -866,6 +916,8 @@ static int conf_init (flux_plugin_t *p, struct perilog_conf *conf)
     json_t *log_ignore = NULL;
     int prolog_per_rank = 0;
     int epilog_per_rank = 0;
+    const char *prolog_timeout = "30m";
```

Comment:
Is there a way to disable the prolog timeout?  Seems like setting it to 0 would cause it to trigger immediately.  Maybe that should be special cased to "disabled"?

Reply:
The timer watcher is not created unless `timeout > 0.0`:
```c
    if (timeout > 0.0) {
        flux_watcher_t *w;
        if (!(w = flux_timer_watcher_create (flux_get_reactor (h),
                                             timeout,
                                             0.,
                                             timeout_cb,
                                             proc))) {
```
So setting it to 0 disables the timer. You are suggesting that the literal string "disabled" should be a synonym for `0`?

Oh, I guess since `fsd_parse_duration()` is used, `inf`, `infinity` or `INF` would also work. However, this would have to be special cased in the stats, so maybe I'll just make Infinity and alias for 0, i.e. disabled (if that helps)

Reply:
0 is fine.  I must have misread the code.   It seemed like 0 went straight into the timer watcher `after` value which means time out immediately